bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/api/creators/ValueCreator.java

```diff
@@ -680,22 +681,47 @@ public static BXml createXmlProcessingInstruction(BString target, BString data,
      *
      * @param mapType map type.
      * @return map value
+     * @deprecated use {@link #createMapValue(MapType)} instead
      */
+    @Deprecated
     public static BMap<BString, Object> createMapValue(Type mapType) {
```

Comment:
There are some places we use this API to create record values without passing the package information (Eg. : cli args). In those cases, deprecating this API will be wrong. Do we need another API as `createRecordValue(RecordType recordType)`?

Reply:
I think so. Hence I added two new methods of `createRecordValue(RecordType recordType ` kind.